src/RF24LogLevel.h

```diff
@@ -49,5 +48,50 @@ enum RF24LogLevel : uint8_t
     /** (`0377` or `0xFF`) the level to enable all log messages (disables filtering of levels) */
     ALL = 0xFF
 };
+/**@} */
+
+/**
+ * @defgroup lvlDesc Level Descriptions
+ * @brief Level descriptions
+ *
+ * These can be modigied using RF24LOG_SHORT_DESC or RF24LOG_TERSE_DESC macros.
+ * @{
+ */
+#if defined(RF24LOG_SHORT_DESC)
+const char RF24LogDescError[] = " ERR";
+const char RF24LogDescWarn[]  = "WARN";
+const char RF24LogDescInfo[]  = "INFO";
+const char RF24LogDescDebug[] = " DBG";
+const char RF24LogDescLevel[] = "L ";
+
+#elif defined(RF24LOG_TERSE_DESC)
+const char RF24LogDescError[] = " E";
+const char RF24LogDescWarn[]  = " W";
+const char RF24LogDescInfo[]  = " I";
+const char RF24LogDescDebug[] = "DB";
+const char RF24LogDescLevel[] = "";
+
+#else // !defined(RF24LOG_*_DESC)
+/** @brief description of the @ref ERROR base level */
+const char RF24LogDescError[] = "ERROR";
+/** @brief description of the @ref WARN base level */
+const char RF24LogDescWarn[] = " WARN";
+/** @brief description of the @ref INFO base level */
+const char RF24LogDescInfo[] = " INFO";
+/** @brief description of the @ref DEBUG base level */
+const char RF24LogDescDebug[] = "DEBUG";
+/** @brief description of the @ref DEBUG base level */
+const char RF24LogDescLevel[] = "Lvl ";
+#endif // !defined(RF24LOG_*_DESC)
+
+/** @brief collection of the base level descriptions */
+const char * const RF24LogDescLevels[] =
+{
+    RF24LogDescError,
+    RF24LogDescWarn,
+    RF24LogDescInfo,
+    RF24LogDescDebug
+};
+

```

Comment:
I would remove this definitions from this file. RF24LogLevel.h should contain only an enum with log levels definitions (this saves RAM). 
Arguments pros:
* String representations of log levels should be defined in log handler: from the abstract point of view of the logging library it is irrelevant here do know what the log level strings looks like. They should be defined in some basic (abstract) log handler; is ```Handler.cpp``` file a better choice?
*  If the end developer decides to use an **empty logger** scenario then those definitions are not needed because nothing gets actually logged. 
* it reduces the memory footprint (mostly RAM) of the core part of the logger library

Reply:
>is Handler.cpp file a better choice?

I was actually thinking the level descriptions should go in their own file since they'll be used by all loggers, but yeah I can move this into the _RF24LogAbstractHandler.cpp_. TBH, I was really just experimenting with alternative (but still uniform width) level descriptions.

Reply:
You can also move them to their own file - no problem with that. I just think they should be removed from ```RF24LogLevel.h ```.